src/components/WwwFrame/LendMenu/TheLendMenu.vue

```diff
@@ -55,6 +63,22 @@ export default {
 		LendListMenu,
 		LendMegaMenu,
 	},
+	created() {
+		if (!this.$isServer) {
+			const { enabled } = getExperimentSettingCached(this.apollo, lendMenuExpKey);
+			if (enabled) {
+				trackExperimentVersion(this.apollo, this.$kvTrackEvent, 'TopNav', lendMenuExpKey);
+				getExperimentSettingAsync(this.apollo, lendMenuExpKey)
```

Comment:
`getExperimentSettingAsync` doesn't include the version assignment. I know that using this didn't work:
```js
const lendMenuCategorySortEXP = this.apollo.readFragment({
	id: 'Experiment:lend_menu_category_sort',
	fragment: experimentVersionFragment,
}) || {};
```
I think that didn't work because the experiment assignment query wasn't called. Instead of using `readFragment`, I would recommend adding this to `lendMenuData.graphql`:
```gql
experiment(id: "Experiment:lend_menu_category_sort") @client {
	id
	version
}
```
Which will make the experiment version available in the `onLoad` function, allowing the experiment flag to be set before the categories are set:
```js
this.apollo.watchQuery({ query: publicLendMenuQuery }).subscribe({
	next: ({ data }) => {
		const version = _get(data, 'experiment.version');
		const { enabled } = getExperimentSettingCached(this.apollo, lendMenuExpKey);
		if (enabled) {
			trackExperimentVersion(this.apollo, this.$kvTrackEvent, 'TopNav', lendMenuExpKey);
			this.weighedCategoriesExp = version === 'b';
		}
		this.categories = _get(data, 'lend.loanChannels.values');
		this.isChannelsLoading = false;
	}
});
```